internal/criocli/criocli.go

```diff
@@ -248,6 +248,9 @@ func mergeConfig(config *libconfig.Config, ctx *cli.Context) error {
 	if ctx.IsSet("manage-ns-lifecycle") {
 		config.ManageNSLifecycle = ctx.Bool("manage-ns-lifecycle")
 	}
+	if ctx.IsSet("drop-infra") {
```

Comment:
Wondering if calling this `drop-infra-container` or `drop-infra-ctr` is better or leave it as is.

Reply:
yeah `drop-infra-ctr` is more clear... I may make that change on next rebase (the only other field in the config that has container or ctr is `ctr_stop_timeout` so we should keep with `ctr` imo)